modules/docs/arrow-docs/docs/docs/effects/fx/polymorphism/README.md

```diff
@@ -26,17 +26,17 @@ import arrow.effects.extensions.io.fx.fx
 suspend fun printThreadName(): Unit =
   println(Thread.currentThread().name)
 
-/* for all `F` that provide an `Fx` extension define a program function
+/* for all `F` that provide an `Fx` extension define a program function */
 fun <F> Fx<F>.program(): Kind<F, Int> =
-  fx { !effect { sideEffect() } }
+  fx { !effect { printThreadName() } }
 
-/* for all `F` that provide an `UnsafeRun` extension define a main function
+/* for all `F` that provide an `UnsafeRun` extension define a main function */
 fun <F> UnsafeRun<F>.main(fx: Fx<F>): Int =
   unsafe { runBlocking { fx.program() } }
 
 /* Run program in the IO monad */
 fun main() =
-  IO.unsafeRun().main(IO.fx()) 
+  IO.unsafeRun().main(IO.fx()) //XXX This doesn't compile
```

Comment:
There seems to be no extension function `fx()`.

There's a similar example in `EffectsSuspendDSLTests -> "FX supports polymorphism"` with the import `import arrow.effects.extensions.io.fx.fx` which cannot be resolved

Reply:
@dominv is this because the remote server is not including the right libs?

Reply:
Sounds like that, because the import for `arrow.effects.extensions.io.fx.fx` is not properly resolved on my machine as well.